varya/assets/css/variables-editor.css

```diff
@@ -90,7 +90,7 @@
 	--button--color-text-active: var(--button--color-text);
 	--button--color-background: var(--global--color-primary);
 	--button--color-background-hover: var(--global--color-primary-hover);
-	--button--color-active: var(--global--color-primary);
+	--button--color-background-active: var(--global--color-foreground);
```

Comment:
@jffng This just makes the active colors specific to the background-color, similar to the `--button--color-background-hover` variable.